### PR TITLE
how-to/deploy: increase the usage precautions for labels

### PR DESCRIPTION
**Pull request by superlzs0476**
### What is changed, added or deleted? 
deploy:Increase the usage precautions for labels.

### What is the related PR or file link(s)? 
N/A

### Which version does your change affect? 
v3.0/v2.1/dev

### CODE REVIEW

**Comment by superlzs0476** [comment]
@lilin90 @nolouch  PTAL

File: v3.0/how-to/deploy/geographic-redundancy/location-awareness.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ location-labels = ["zone", "rack", "host"]
 
 其中 `location-labels` 需要与 TiKV 的 `labels` 名字对应，这样 PD 才能知道这些 `labels` 代表了 TiKV 的拓扑结构。
 
+> **注意：**
+>
+> PD 的 `location-labels` 配置必须和 TiKV 的 `labels` 配置一同设置，它们之间的且的关系，否则 `labels` 不会生效。
```
Comment (lilin90) [inline v3.0/how-to/deploy/geographic-redundancy/location-awareness.md]:
```suggestion
> 必须同时配置 PD 的 `location-labels` 和 TiKV 的 `labels` 参数，否则 `labels` 不会生效。
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -457,6 +457,7 @@ mount -t ext4
 172.16.10.2
 172.16.10.3
 
+# 注意： TiKV 的 labels 的使用，必须配合 PD 的 location_labels 参数，否则 labels 设置不生效。
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
# 注意：要使用 TiKV 的 labels，必须同时配置 PD 的 location_labels 参数，否则 labels 设置不生效。
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 3]
```diff
@@ -487,6 +488,7 @@ TiKV3-2 ansible_host=172.16.10.6 deploy_dir=/data2/deploy tikv_port=20172 labels
 172.16.10.5
 172.16.10.6
 
+# 注意：配合 labels 使用，集群部署阶段必须设置 location_labels 参数。
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
# 注意：为使 TiKV 的 labels 设置生效，部署集群时必须设置 PD 的 location_labels 参数。
```

**Comment by lilin90** [comment]
@superlzs0476 Please pay attention to your PR title and commit messages. Follow https://github.com/pingcap/community/blob/master/commit-message-pr-style.md

**Comment by lilin90** [comment]
@dcalvin Please help update the changes to pingcap/docs. Note that the changes apply to `dev`, `v3.0` and `v2.1`. `dev` and `v2.1` are updated in #1929.

**Pull request merged**